k-distribution/include/builtin/domains.k

```diff
@@ -830,11 +830,11 @@ module BYTES-IN-K
   rule replaceAtBytes(B : BS, N, BS') => B : replaceAtBytes(BS, N -Int 1, BS') requires N >Int 0
   rule replaceAtBytes(_ : BS, 0, B : BS') => B : replaceAtBytes(BS, 0, BS')
 
-  syntax Bytes ::= padRightBytes(Bytes, Int, Int) [function]
-                 | padLeftBytes(Bytes, Int, Int) [function]
+  syntax Bytes ::= padRightBytes(Bytes, Int, Int) [function, klabel(padRightBytes)]
```

Comment:
I don't understand this change; why was it necessary?

Reply:
I don't remember. Probably for debugging. Can be removed.

Reply:
I'd rather we remove it because as it stands it's not going to do anything, which makes it redundant.